From f69910d67fa2a8844d6cde7623d4273d36c77f23 Mon Sep 17 00:00:00 2001
From: Adrian DC <radian.dc@gmail.com>
Date: Mon, 24 Jul 2017 16:41:53 +0200
Subject: [PATCH 01/14] libhealthd: Add android-base and minui include paths

 * Required to access android-base/unique_fd.h

 * Includes following commit from CAF device/qcom/common

healthd: Add include path for minui.h
The minui.h header file path has changed such that it
now resides within recovery/minui/include. Update the
healthd makefile to reflect this
Change-Id: I4ed371156b075f5b96404480b3703f113003979b

Change-Id: I60bc31ecaa07dad40b37265ded37d64b492bd029
---
 charger/Android.mk | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/charger/Android.mk b/charger/Android.mk
index cde351b..5d42e6a 100644
--- a/charger/Android.mk
+++ b/charger/Android.mk
@@ -20,7 +20,8 @@ LOCAL_MODULE := libhealthd.lineage
 LOCAL_CFLAGS := -Werror
 LOCAL_C_INCLUDES := \
     system/core/healthd/include \
-    bootable/recovery
+    system/core/base/include \
+    bootable/recovery/minui/include
 ifneq ($(BACKLIGHT_PATH),)
     LOCAL_CFLAGS += -DHEALTHD_BACKLIGHT_PATH=\"$(BACKLIGHT_PATH)\"
 endif
-- 
2.7.4

